packages/cli/internal/pkg/cli/workflow/manager.go

```diff
@@ -663,6 +666,22 @@ func (m *Manager) updateInProject(instance *InstanceSummary) {
 	_, instance.InProject = m.projectSpec.Workflows[instance.WorkflowName]
 }
 
+func (m *Manager) setRequest(instance *InstanceSummary) {
+	if m.err != nil || instance == nil || instance.Request != "" {
```

Comment:
If `instance` is already null or `Request` already set, then there's probably something wrong in our code. Rather than silently fail to set the request, we should at the very least log a warning that something is wrong.